pkg/ovsdb/transact.go

```diff
@@ -1045,11 +1048,11 @@ func doUpdate(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.O
 		ok = txn.schemas.Validate(txn.request.DBName, ovsOp.Table, &ovsOp.Row)
 		if !ok {
 			err := errors.New(E_CONSTRAINT_VIOLATION)
-			ovsResult.Error = err.Error()
+			*ovsResult.Error = err.Error()
 			return err
 		}
 
-		ovsResult.Count = ovsResult.Count + 1
+		*ovsResult.Count++
 		err = RowUpdate(tableSchema, row, ovsOp.Row)
 		if err != nil {
 			return err
```

Comment:
you just return errors, in each operation, but you have to set the Erro filed of the specific operation

Reply:
ok